### PR TITLE
denote required node version in readme

### PR DESCRIPTION
**Pull request by mrjones-plip**
per #medic/couch2pg/issues/31

### CODE REVIEW

**Review by kitsao — APPROVED** [review]
Looks good.

**Comment by mrjones-plip** [comment]
Thanks @kitsao ! 

@garethbowen  - it looks like @kitsao  doesn't have write access to this repo so the "lgtm" doesn't count I guess? Doesn't seem correct, but maybe there's some history about perms to this repos I'm not aware of.

**Review by garethbowen — APPROVED** [review]
LGTM2

**Pull request merged**
